src/longRunningCalls/longrunning.ts

```diff
@@ -234,6 +295,60 @@ export class Operation extends EventEmitter {
     }
   }
 
+  _unpackComputeResponse(
+    op: ComputeLROOperation,
+    callback?: GetOperationCallback
+  ) {
+    if (this._isComputeOperationDone(op)) {
+      if (op.error) {
+        const err = fromComputeOperationError(op);
+        if (callback) {
+          return callback(err);
+        }
+        return;
+      }
+      if (op) {
+        this.response = op;
+        this.done = true;
+      }
+    }
+    if (callback) {
+      callback(null, this.response!, null, op);
+    }
+  }
+
+  _isComputeOperationDone(op: ComputeLROOperation): boolean {
```

Comment:
This method should either be static or be defined outside of the class since it never accesses `this`.